pkg/util/provider/drain/drain.go

```diff
@@ -635,6 +635,7 @@ func (o *Options) checkAndDeleteWorker(volumeAttachmentEventCh chan *storagev1.V
 }
 
 func (o *Options) evictPodsWithPVInternal(
```

Comment:
I suspect this function or it's wrapping function has a mainContext created and passed around. With this change now, we can rather use this background context to derive the context?